libs/@csnx/design-tokens/tokens/index.js

```diff
@@ -143,7 +143,6 @@ export const tokens = {
 			100: '#3E3323',
 			200: '#574835',
 			300: '#6B5840',
-			/** Deprecated: Use colour.culture.400 */
```

Comment:
this isn't good :(

Reply:
Should this be `/** @deprecated use colour.culture[400] */`